src/resmom/mom_hook_func.c

```diff
@@ -1175,8 +1175,9 @@ run_hook(hook *phook, unsigned int event_type, mom_hook_input_t *hook_input,
 		}
 
 #ifdef	WIN32
-		secure_file(hook_inputfile, "Administrators",
```

Comment:
Please verify all the secure_file() call in this file and log the file and username etc
https://github.com/PBSPro/pbspro/blob/73906777864cefafbeb803116751b54ac0fdf2f5/src/resmom/mom_hook_func.c#L1125